### PR TITLE
DCKUBE-613: Configurable grace periods

### PR DESCRIPTION
**Pull request by tarka**
I couldn't find the graceful periods for Crowd and Jira, so went with the Kubernetes default.

### CODE REVIEW

**Review by errcode1202 — APPROVED** [review]

**Review by errcode1202 — CHANGES_REQUESTED** [review]
Hey Steve, you need to regenerate the docs so that the new property `terminationGracePeriodSeconds` is rendered as part of them, see: https://github.com/atlassian-labs/data-center-helm-charts/blob/1e8b150023ab419e0ac8d11be3c9babbc50f45d2/DEVELOPMENT.md

File: src/main/charts/bitbucket/values.yaml
Comment (errcode1202) [inline src/main/charts/bitbucket/values.yaml:39]: Not sure if this sentence reads correctly: `should be set the Bitbucket internal grace period ` should it be `should will set the Bitbucket internal grace period ` ? 

This needs to be fixed for the other products too.

File: src/main/charts/bitbucket/templates/statefulset.yaml
Comment (errcode1202) [inline src/main/charts/bitbucket/templates/statefulset.yaml:28]: Should we add tests to ensure that the correct grace period (whats defined in `values.yaml`) is applied?

Comment (tarka) [reply src/main/charts/bitbucket/templates/statefulset.yaml:28]: This is done via the expected-output tests.

File: src/main/charts/bitbucket/values.yaml
Comment (tarka) [reply src/main/charts/bitbucket/values.yaml:39]: Bitbucket has it's own internal grace-period for SSH etc, which the customer can override. This setting is for the Kubernetes grace setting, which is separate. The Kubernetes one should be slightly higher than the BB one, so that BB can complete it's shutdown before the Kubernetes grace period times out.

But there is a missing word, I'll fix that.

**Review by nghazali — APPROVED** [review]

File: src/main/charts/confluence/templates/statefulset-synchrony.yaml
Comment (badgersow) [inline src/main/charts/confluence/templates/statefulset-synchrony.yaml:24]: Just to let you know - I'm working on a similar PR in #250 and I will likely move this property to different value (because I also want to add shutdown command). I will wait until you merge and then start on top of your changes.

**Review by badgersow — APPROVED** [review]

File: src/main/charts/confluence/templates/statefulset-synchrony.yaml
Comment (tarka) [reply src/main/charts/confluence/templates/statefulset-synchrony.yaml:24]: :+1:

**Review by errcode1202 — APPROVED** [review]

File: src/main/charts/bitbucket/README.md
Comment (badgersow) [inline src/main/charts/bitbucket/README.md:50]: Any ideas how so much additional `object` documentation lines were created? My pre-commit hook doesn't generate them.

**Pull request merged**
